cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -44,6 +44,7 @@ spec:
     # Mons should only be allowed on the same node for test environments where data loss is acceptable.
     allowMultiplePerNode: false
   mgr:
+    count: 1
```

Comment:
- We would need a comment here that gives an idea about when it should be changed.
- Also document the setting in ceph-cluster-crd.md